packages/@react-spectrum/list/stories/ListView.stories.tsx

```diff
@@ -319,6 +319,36 @@ storiesOf('ListView', module)
           (item) => <Item textValue={item.title}><Image src={item.url} /><Text>{item.title}</Text><Text slot="description">JPG</Text></Item>
         }
     </ListView>
+  ))
+  .add('overflowMode="truncate" (default)', () => (
+    <ListView width="250px" overflowMode="truncate">
+      <Item textValue="row 1">row 1 with a very very very very very long title</Item>
+      <Item textValue="row 2">
+        <Text>Text slot with a really really really long name</Text>
+        <Text slot="description">Description slot with a really really long name</Text>
+      </Item>
+      <Item textValue="row 3">
+        <Content>Content slot with really really long name</Content>
+      </Item>
+      <Item textValue="row 4">
+        <Link >Link slot with a very very very very long name</Link>
+      </Item>
```

Comment:
Can delete these two rows since Link and Content shouldn't be in the ListView anymore

Reply:
yeah, was a bad merge, didn't notice the change